biocypher_metta/adapters/epd_adapter.py

```diff
@@ -0,0 +1,65 @@
+import csv
+import gzip
+import pickle
+from biocypher_metta.adapters import Adapter
+# Example EPD bed input file:
+# chr1 959245 959305 NOC2L_1 900 - 959245 959256
+# chr1 960583 960643 KLHL17_1 900 + 960632 960643
+# chr1 966432 966492 PLEKHN1_1 900 + 966481 966492
+# chr1 976670 976730 PERM1_1 900 - 976670 976681
+
+class EPDAdapter(Adapter):
+    INDEX = {'chr' : 0, 'coord_start' : 1, 'coord_end' : 2, 'gene_id' : 3}
+
+    def __init__(self, filepath, hgnc_to_ensembl_map, write_properties, add_provenance, type='propmoter', label='promoter', delimiter=' '):
```

Comment:
There is a typo in the `type` argument's default value. I think you meant to write `promoter`?